src/App.tsx

```diff
@@ -1,61 +1,193 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
 import './App.scss';
+import 'bulma/css/bulma.css';
+
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todosInitial: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+let nextTodoId = Math.max(...todosInitial.map(todo => todo.id)) + 1;
```

Comment:
Consider using `useState` hook for `nextTodoId` variable to avoid global state. 💡